libqtile/confreader.py

```diff
@@ -99,6 +99,25 @@ def update(self, *, fake_screens=None, **settings):
                 value = getattr(self, key, default[key])
             setattr(self, key, value)
 
+    def reload_config_submodules(self, path):
+        """Reloads python files from same folder as config file."""
+        to_reload = []
+        folder = path.parent
+        for name, module in sys.modules.items():
+
+            # Skip built-ins and anything with no filepath.
+            if getattr(module, "__file__", False):
+                subpath = Path(module.__file__).parent
+
+                # Check if the module is in the config folder or subfolder
+                if folder in subpath.parents:
```

Comment:
Just trying to understand the logic here, maybe I'm being slow. `folder` is the parent folder of the config file, e.g. `~/.config/qtile`. If `module` refers to the file at `~/.config/qtile/config.py`, and `subpath` is its parent, which is `~/.config/qtile`. So `subpath.parents` will be `~/.config` and above, right? Am I misreading something?

Reply:
I may have one too many `parents` here!

Yes - `folder` is the folder that the `config.py` file is in. I need to remove the `.parent` from line 110 as the `.parents` in 113 will then be the folder for the submodules and all the parent. Therefore, assuming the config file is in the same folder, or parent folder, of the submodules, it should match the test in line 113.